wot-wg-2023-draft.html

```diff
@@ -284,6 +284,24 @@ <h3>
           The Working Group will deliver the following W3C normative specifications:
         </p>
         <dl>
+
+
+          <dt id="wot-arch-update" class="spec">Web of Things (WoT) Architecture (Update)</dt>
```

Comment:
Suggested change to align with "Generic Names" in Pr #79 
```suggestion
          <dt id="wot-arch-update" class="spec">Architecture (Update)</dt>
```